pubsublite/integration_test.go

```diff
@@ -189,6 +189,7 @@ func TestIntegration_ResourceAdminOperations(t *testing.T) {
 
 	topicUpdate1 := TopicConfigToUpdate{
 		Name:                       topicPath,
+		PartitionCount:             2,
```

Comment:
What happens when a `PartitionCount` doesn't change? Will the server throw an error here?

Reply:
Good question. I just tried it and it's a no-op.